### PR TITLE
Image example, readme for UI, sagemaker example update

### PR DESCRIPTION
**Pull request by riteshkarval**

### CODE REVIEW

File: image_cloudevents/README.md [hunk 1]
```diff
@@ -1 +1,96 @@
...
+## Section 3: Modelmonitoring
+DKube provides Python SDK for creating a modelmonitor programmatically. You could also choose to create a modelmonitor from the DKube UI. This example cuurently support creating monitor using SDK. 
+
+- **Create Modelmonitor using SDK**
+1. From **workspace/chest-xray/image_cloudevents** open [modelmonitor.ipynb](https://github.com/oneconvergence/dkube-examples/tree/monitoring/image_cloudevents/modelmonitor.ipynb) and run all the cells. New model monitor will be created.
+2. Predict and Groundtruth dataset data will be generated by Data Generation step and will be utilised by modelmonitor.
+
+## Section 4: Data Generation
+1. Open [data_generation.ipynb](https://github.com/oneconvergence/dkube-examples/tree/monitoring/image_cloudevents/data_generation.ipynb) notebook for making predictions with the deployemnt endpoint and generate groundtruth datasets.
+2. In 1st cell, Update Frequency according to what you set in Modelmonitor for Drift. For eg: for 5 minutes, specify it as `5m` and to specify the frequency in hours use `5h` for 5 hours interval.
```
Comment (songole) [inline image_cloudevents/README.md]: There is no run frequency in the 1st cell, right? It is in resources.ipynb. You may want to move this line up there.

Also, it is not 5m, it appears it expects always in min? or, is it the default?

I fixed the steps for insurance_datasources.

File: image_cloudevents/README.md [hunk 2]
```diff
@@ -1 +1,96 @@
...
+2. The pipeline includes preprocessing, training and serving stages. Run all cells
+     - **preprocessing**: the preprocessing stage generates the dataset (either training-data or retraining-data) depending on user choice.
+     - **training**: the training stage takes the generated dataset as input, train a sgd model and outputs the model.
+     - **serving**: The serving stage takes the generated model and serve it with a predict endpoint for inference.
+3. Verify that the pipeline has created the following resources
+     - Datasets: 'insurance-training-data' with version v2.
+     - Model: 'insurance-model' with version v2
+
+## Section 3: Modelmonitoring
+DKube provides Python SDK for creating a modelmonitor programmatically. You could also choose to create a modelmonitor from the DKube UI. This example cuurently support creating monitor using SDK.
```
Comment (songole) [inline image_cloudevents/README.md:79]: Is there an image example with UI steps for creating modelmonitor? We need to have at least 1.

I see there is README.ui.md. What is it for?

Comment (riteshkarval) [reply image_cloudevents/README.md:79]: The readme.ui.md is having steps to make model monitor from UI, as we are doing for other examples.

**Comment by riteshkarval** [comment]
@songole the changes are added. can we merge it?

**Pull request merged**